include/libopencm3/stm32/l4/rcc.h

```diff
@@ -587,7 +587,7 @@ Twelve frequency ranges are available: 100 kHz, 200 kHz, 400 kHz, 800 kHz,
 #define RCC_CCIPR_ADCSEL_MASK		0x3
 #define RCC_CCIPR_ADCSEL_SHIFT		28
 
-#define RCC_CCIPR_CLK48SEL_NONE		0
+#define RCC_CCIPR_CLK48SEL_HSI48	0
```

Comment:
is this actually a problem with different L4s? looks suspect

Reply:
I checked it against both reference manuals I found for the L4 (RM0394 and RM0351)

Reply:
Ok, found it.  I wrote that from RM0351 rev1, this bit is listed has having been updated in revision 5, the current revision :) so yeah, looks good now :)